src/quill.imageDrop.ts

```diff
@@ -90,6 +102,27 @@ export class ImageDrop {
     await this.pasteFilesIntoQuill(imageFiles);
   }
 
+  private async processHtml(html: string) {
+
+    const tmp = document.createElement('div')
+    tmp.innerHTML = html;
+    const imgNodeList = tmp.querySelectorAll('img');
+
+    for (let i = 0; i < imgNodeList.length; i++) {
+      const img = imgNodeList[i];
+      if (img.src.startsWith("http")) {
+        const blob = await (await fetch(img.src)).blob()
+        img.src = await file2b64(blob);
+      }
+      img.src = await this.processImage(img.src);
+    }
```

Comment:
This is pretty clever! 🥇 However there could be `CORS`, or `401` errors during fetching, that would crash this whole `processHtml` function.

I think a good solution to these errors would be to add a placeholder image, instead of failing completely, WDYT?

Something like this maybe (encoded as a tiny base64 string and included in the library)

![](https://placehold.co/200?text=Image%20Unavailable&font=roboto)